### PR TITLE
[ADD] patched the phantom error log, no longer print of the entire DOM

### PR DESCRIPTION
**Pull request by oscarolar**

### CODE REVIEW

**Comment by nhomar** [comment]
It is nice the concept, just some improvements I see are necessary:
1. Naming the module disable_log sound you will disable everything, I prefer a name like "web_tour_no_html" which is more (it is a proposal you can decide another, but closest to what it does).
2. Can you put in the README a pair of Print Screens with:
   a. Normal error (that will be the before).
   b. Error silenced (that will be the after).
3. I can't read in the README a usage section ([see the readme template](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)), even if reading the code is pretty obvious, I feel it is better if we explain the obvious part.
4. What's the order in which it must be installed?, because until the template is installed I think this will overwrite the template, then the order matters, how do you expect it works?.
5. Should not we add an +    'auto_install': True?, if yes, it is better an specific repository for this module only in order to simply declare it in the CI and not affect the production environments with an installed module just with a declaration.

Inline comment in the code.

**Comment by moylop260** [comment]
> @nhomar 
> 5. Should not we add an + 'auto_install': True?, if yes, it is better an specific repository for this module only in order to simply declare it in the CI and not affect the production environments with an installed module just with a declaration.

FYI [where-CI-set-auto-install-True](https://github.com/Vauxoo/maintainer-quality-tools/blob/60944f409262920e7d75b5ab455c7932e10ff42f/travis/travis_install_nightly#L91-L111)

File: tour_disable_log/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# coding: utf-8
+###########################################################################
```
Comment (moylop260) [inline tour_disable_log/__openerp__.py]: Please, Use [short license format](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3)

**Comment by nhomar** [comment]
On Fri, Jun 17, 2016 at 1:24 PM, moylop260 notifications@github.com wrote:

> where-set-auto-install-True

Roger that:

Oscar: Can you make a proper PR where @moylop260  is doing that job after fix my
comments and merge it please.

## 

Nhomar Hernandez
CEO Vauxoo.
Site: http://vauxoo.com
Twitter: @nhomar
Blog: http://nhomar.com
Github User: https://github.com/nhomar
Odoo Gold Partner
Skype: nhomar00 (Envia mail previo no lo superviso siempre).
HangOut: nhomar@vauxoo.com
Móvil Venezuela:
+58 4144110269
Móvil México:
+52 1 4773933942

File: tour_disable_log/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+#    Module Writen to OpenERP, Open Source Management Solution
+#
+#    Copyright (c) 2015 Vauxoo - http://www.vauxoo.com/
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+############################################################################
+#    Coded by: luisll (luisll@vauxoo.com) and others
```
Comment (moylop260) [inline tour_disable_log/__openerp__.py]: `s/luisll/oscar/g`

File: tour_disable_log/static/src/js/tour_disable_log.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+(function () {
+    'use strict';
+    var error = function (step, message) {
+            var state = openerp.Tour.getState();
```
Comment (moylop260) [inline tour_disable_log/static/src/js/tour_disable_log.js]: Please, add a comment with the original path code that you used for copy&paste

**Comment by oscarolar** [comment]
> What's the order in which it must be installed?, because until the template is installed I think this will overwrite the template, then the order matters, how do you expect it works?.

It depends from web app, it has the parent template.

**Pull request merged**